src/runtime/runtime.ts

```diff
@@ -61,6 +68,19 @@ const lazyLoadSpecLocation = async (location: Fig.SpecLocation): Promise<Fig.Spe
   return; //TODO: implement spec location loading
 };
 
+export const loadLocalSpecsSet = async () => {
+  const specsPath = getConfig()?.specs?.path;
+  if (!specsPath) {
+    return;
+  }
+  try {
+    const { default: speclist, diffVersionedCompletions: versionedSpeclist } = await import(path.join(specsPath, "index.js"));
```

Comment:
Can you add a check to make sure that the directory exists? If it doesn't, add a debug log and just continue to the next possible spec import

Reply:
<img width="895" alt="image" src="https://github.com/microsoft/inshellisense/assets/9245110/0332491d-d298-47d5-89a1-5a5b11685176">
I think if the direct import fails, it is sufficient to handle the exception, and there is no need to separately check the folder.